.github/workflows/validations.yml

```
@@ -11,93 +11,183 @@ jobs:
+      - name: Enable PostgreSQL extensions
+        env:
+          RAILS_ENV: test
+          DB_HOST: localhost
+          DB_USER: root
+          DB_PASS: password
         run: |
-          bundle exec rubocop
-      - name: Run tests
+          bundle exec rails runner "ActiveRecord::Base.connection.execute('CREATE EXTENSION IF NOT EXISTS pg_trgm')"
+          bundle exec rails runner "ActiveRecord::Base.connection.execute('CREATE EXTENSION IF NOT EXISTS unaccent')"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)